script.js

```diff
@@ -396,38 +396,20 @@ function rotateJ(values) {
   }
 }
 
-function checkJRotationClockwise(){
+function checkJRotation(clockwise){
   if (curBlock[0].xPos <= curBlock[3].xPos && curBlock[0].yPos <= curBlock[3].yPos) {
-    return 0;
+    return clockwise ? 0 : 2;
```

Comment:
This thing with the ? and : is an operator you may not have seen before called the ternary operator. It's kind of like an if except its an expression that produces a value so this line will return 0 if clockwise is true and 2 if it's false.